### PR TITLE
python3Packages.icontract: 2.6.2 -> 2.6.3

### PR DESCRIPTION
**Pull request by thiagokokada**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by thiagokokada** [comment]
Result of `nixpkgs-review pr 258554` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>python310Packages.icontract</li>
    <li>python310Packages.icontract.dist</li>
    <li>python310Packages.pylddwrap</li>
    <li>python310Packages.pylddwrap.dist</li>
    <li>python311Packages.icontract</li>
    <li>python311Packages.icontract.dist</li>
    <li>python311Packages.pylddwrap</li>
    <li>python311Packages.pylddwrap.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/icontract/default.nix [hunk 1]
```diff
@@ -52,12 +52,6 @@ buildPythonPackage rec {
     "tests/test_mypy_decorators.py"
   ];
 
-  # Upstream adds some plain text files direct to the package's root directory
-  # https://github.com/Parquery/icontract/blob/master/setup.py#L63
-  postInstall = ''
-    rm -f $out/{LICENSE.txt,README.rst,requirements.txt}
-  '';
```
Comment (thiagokokada) [inline pkgs/development/python-modules/icontract/default.nix:59]: Fixed in https://github.com/Parquery/icontract/pull/269.

**Comment by natsukium** [comment]
@ofborg build python310Packages.icontract python311Packages.icontract

**Comment by fabianhjr** [comment]
Missing hash update?

**Review by fabianhjr — CHANGES_REQUESTED** [review]
Missing hash update?

**Comment by thiagokokada** [comment]
> Missing hash update?

Good catch. Fixed.

**Comment by fabianhjr** [comment]
Result of `nixpkgs-review pr 258554` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>python310Packages.icontract</li>
    <li>python310Packages.icontract.dist</li>
    <li>python310Packages.pylddwrap</li>
    <li>python310Packages.pylddwrap.dist</li>
    <li>python311Packages.icontract</li>
    <li>python311Packages.icontract.dist</li>
    <li>python311Packages.pylddwrap</li>
    <li>python311Packages.pylddwrap.dist</li>
  </ul>
</details>

**Review by fabianhjr — APPROVED** [review]

**Pull request merged**